slippymap/slippymap.go

```diff
@@ -309,7 +296,7 @@ func (sm *SlippyMap) makeTile(osmX, osmY, offsetX, offsetY int) {
 
 	go func() {
 		// get tile artwork
-		tilePath, err := cacheTile(osmX, osmY, sm.zoomLevel, sm.pathTileCache)
+		tilePath, err := cacheTile(t.osmX, t.osmY, t.zoomLevel, sm.pathTileCache)
```

Comment:
I _think_ this might be why the wrong tiles were loading. Between when `makeTile` is called and when this goroutine starts, `sm.zoomLevel` could have changed - a very small window for race condition but a window nonetheless. 

I'm not _sure_ that this is the cause of the bug. I'm having trouble reproducing the bug today. I _think_ if this bug were the case, the tile would be filtered out for being at the wrong zoom level. _Maybe_ it was because my laptop trackpad was zooming in and out again and `sm.zoomLevel` was captured in the middle of that.

Either way, I feel like this change is a technically correct change, even if it's not the source of the bug I was seeing.

Reply:
Technically correct is the best kind of correct. :-)